index.md

```diff
@@ -15,16 +15,25 @@ The course will take place on the **4th** and the **5th of February 2020** (duri
 Registration can be done via the following link: *to be determined*.
 
 Admission prices are:
-- **for UGent doctoral school members**: EUR 50 (paid by the Doctoral School program);
-- **external members**: EUR 250.
+- **for UGent doctoral school members**: free of charge;
```

Comment:
- **for UGent members (not members of the Doctorial School)**: EUR 50

Reply:
check